src/main/java/com/github/hey_world_team/currency_converter/model/chain/Trade.java

```diff
@@ -0,0 +1,24 @@
+package com.github.hey_world_team.currency_converter.model.chain;
+
+import com.github.hey_world_team.currency_converter.service.currency.CurrencyType;
+import lombok.*;
+import lombok.extern.slf4j.Slf4j;
+
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@ToString
+@Slf4j
+@Builder
+public class Trade {
+    private long id;
+    private String coinICC;
+    private CurrencyType currency;
+    private BigDecimal price;
+    private boolean isBuy;
```

Comment:
We have same variables "isBuy" and  "isSell" here and in Order Class.
Let's delete one from them.

Reply:
Yes, you're right